src/Guide/Api/Guider.hs

```diff
@@ -0,0 +1,39 @@
+{- | Guider monad with Config to replace servant`s Handler. -}
```

Comment:
How does it keep happening that you use `` ` `` instead of `'`?

```suggestion
{- | 'Guider' monad with 'Config' to replace Servant's 'Handler'. -}
```

Reply:
Also, I'm not sure whether this will work in Haddock, it's safer to move this comment to be directly before the module declaration.